core/src/main/java/org/apache/cxf/io/DelayedCachedOutputStreamCleaner.java

```diff
@@ -79,7 +79,8 @@ default void forceClean() {
     }
 
     private static final class SingleTimerDelayedCleaner implements DelayedCleaner {
-        private static final Timer TIMER = new Timer("DelayedCachedOutputStreamCleaner", true);
+        private static volatile Timer timer;
```

Comment:
Please let's keep static capitalized (conventional):

```suggestion
        private static volatile Timer TIMER;
```

Reply:
Cannot argue with that :-D thank you